src/neo4clj/query_builder.clj

```diff
@@ -100,7 +108,7 @@
   and returns any aliases specified in the representation"
   [{:keys [lookups nodes relationships returns]}]
   (str/join " " (concat
-                 (map #(lookup-query % false) lookups)
+                 (map #(lookup-node % false) lookups)
```

Comment:
Nothing new that you introduced in this PR, but some responsibility seems to off here, or overlapping to be correct.

`create-graph-query` can control `"RETURN"` which means it has to ALWAYS call `lookup-node` with `false`. So now there is two functions that control `"RETURN"`. I don't think this should be in two places at least not chained.

Reply:
This is because the way the graph query works. In all other instances we work on single nodes or relationships, but in graph we need to concat all the different parts together, and to do this we need to ensure the return parts are not part of the generated cypher strings.

Reply:
Then call the `cypher/lookup-node` directly instead.

Reply:
Those two functions have different formats. The one in query_builder.clj returns a complete MATCH string, where the one in cypher.clj returns the different parts of the query without the different protected words such as MATCH, WHERE and RETURN